custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -202,7 +202,7 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
                         for (var fillKey in this.options.fills) {
                             if (fillKey === 'defaultFill') continue;
                             html.push(
-                                '<div class="row no-margin" style="margin-bottom: 5px;">',
+                                '<div class="row no-margin map-legend-color-row">',
```

Comment:
It's a little weird to be applying a margin to an element that also uses a class called `no-margin`, but you didn't introduce that.

Reply:
on the scale of weird things in dashboard front end I'd rate this about a 4/10. :stuck_out_tongue_winking_eye:

Reply:
fair